libkbfs/block_journal_test.go

```diff
@@ -146,7 +150,7 @@ func TestBlockJournalRemoveReferences(t *testing.T) {
 
 	// Remove references.
 	liveCounts, err := j.removeReferences(
-		ctx, map[BlockID][]BlockContext{bID: {bCtx, bCtx2}}, true)
+		ctx, map[BlockID][]BlockContext{bID: {bCtx, bCtx2}}, false)
```

Comment:
Needed because otherwise the `readJournal()` call in `shutdown()` can't calculate the block bytes, if the underlying block is removed.